### DISCUSSION TITLE
Q_Li=2563.8190 Ah is outside the range of possible values [-0.0000, -0.1526]

### DISCUSSION THREAD

**Discussion by Notslytherin1** [pybamm-team/PyBaMM#3179] [General]
ValueError: Q_Li=2563.8190 Ah is outside the range of possible values [-0.0000, -0.1526]
I'm facing this issue in the context of parameters identification of the DFN model using genetic algorithm(pygad).
This error occur when the algorithm try to find the optimal parameters and it shows this error in the first iteration.
i'm working with the following version of pybamm: 23.4.1.
And i wonder how to fix the bounds and how to define their bounds knowing that (Qli, x,y) doesn't exist in the parameter values.
Thank for considering my demand.

**Comment by brosaplanella** [comment]
The capacity seems huge, so probably one of the parameters has been pushed to unrealistic values. Could you share a minimum working example?

**Comment by valentinsulzer** [comment]
If you just don't want to see the error, you could turn off this part of the code by passing `calc_esoh=False` to `sim.solve`. But it's possible that the simulation would just fail somewhere else if the parameters are unfeasible
